automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoModelCopy.scala

```diff
@@ -20,6 +21,94 @@ case class GcsPath(bucketName: GcsBucketName, relativePath: GcsRelativePath) {
   def toUri: String = s"$GCS_SCHEME://${bucketName.name}/${relativePath.name}"
 }
 
+object GcsPath {
```

Comment:
Do we need to copy-paste all of this? Can't we just use workbench-libs instead?

Reply:
+1